content/200-concepts/100-components/01-prisma-schema/07-referential-actions.mdx

```diff
@@ -0,0 +1,327 @@
+---
+title: 'Referential actions'
+metaTitle: 'Referential actions'
+metaDescription: 'Referential actions let you define the update and delete behavior of related models on the database level'
+tocDepth: 2
+preview: true
+---
+
+<TopBlock>
+
+As of **2.26.0** Prisma now supports referential actions as part of the Prisma Schema Language.
+
+This is a [**preview feature**](../../../about/releases#preview) and can be accessed by:
+
+1. [Upgrading to version v2.26.0 of Prisma](../../../guides/upgrade-guides/upgrading-to-latest).
+2. Adding `referentialActions` to the `previewFeatures` field of the [`generator`](../../../reference/api-reference/prisma-schema-reference#generator) block in the `prisma.schema` file.
+
+```prisma file=schema.prisma
+generator client {
+    provider = "prisma-client-js"
+    previewFeatures: ['referentialActions']
```

Comment:
We usually indent with two spaces, don't we?

Reply:
```suggestion
    provider = "prisma-client-js"
    previewFeatures: ["referentialActions"]
```
single quotes don't work I think

Reply:
Copy paste, will change